tree/dataframe/inc/ROOT/RDF/RLoopManager.hxx

```diff
@@ -116,6 +116,8 @@ class RLoopManager : public RNodeBase {
    /// A unique ID that identifies the computation graph that starts with this RLoopManager.
    /// Used, for example, to jit objects in a namespace reserved for this computation graph
    const unsigned int fID = GetNextID();
+   ULong64_t fStartEvt;
+   ULong64_t fEndEvt;
```

Comment:
To avoid that we ever forget to zero-initialize these:

```suggestion
   ULong64_t fEndEntry = 0;
```

(and similarly above, github does not allow multiline code suggestions afaict)